values/jobs/keycloak.gotmpl

```diff
@@ -18,6 +18,8 @@ type: Job
 enabled: true
 runPolicy: Always
 description: Configure OIDC as a primary auhentication method and populate teams to harbor projects
+annotations:
+    policy.otomi.io/ignore: "banned-image-tags"
```

Comment:
can we be precise here and target only the init container?

Reply:
I tired but it did not work :(